plugins/qtAccessionPlugin/modules/accession/templates/indexSuccess.php

```diff
@@ -160,7 +160,7 @@
     <div class="field">
       <h3><?php echo __('Deaccession') ?></h3>
       <div>
-        <?php echo link_to(render_title($item), array($item, 'module' => 'deaccession')) ?>
+        <?php echo link_to($item, array($item, 'module' => 'deaccession')) ?>
```

Comment:
Removing `render_title()` here is changing the behaviour of the application, beyond just removing markdown support, as render_title substitutes an `<em>Untitled</em>` value when the db value is empty.  Is this your intention in removing `render_title()`?

I question whether we need to allow markdown in titles in general or could remove the `render_inline_value()` here: https://github.com/artefactual/atom/blob/qa/2.6.x/lib/helper/QubitHelper.php#L128 ?  I guess markdown can be handy for emphasis in the title in some cases. If we do want to keep markdown support for `render_title()` maybe we need to introduce an option parameter to the function to allow turning off markdown support?

Reply:
Hrm... adding an option to allow turning off Markdown support seems like it might be the way to go. I'll re-roll this PR with that.